src/client/cli/cmd/launch.cpp

```diff
@@ -121,6 +107,7 @@ auto net_digest(const QString& options)
 
     return net;
 }
+
```

Comment:
Why a newline here now?  `clang-format`?

Reply:
No, I don't think so. It just escaped my sight. Removed.